models/experimental/functional_yolov4/tt/ttnn_downsample1.py

```diff
@@ -0,0 +1,57 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+
+import torch
+import torch.nn as nn
+
+import ttnn
+import tt_lib
+from ttnn.model_preprocessing import preprocess_model
+
+
+class TtDownSample1:
+    def __init__(
+        self,
+        parameters,
+    ) -> None:
+        self.c1 = parameters.c1
+        self.c2 = parameters.c2
+        self.c3 = parameters.c3
+        self.c4 = parameters.c4
+        self.c5 = parameters.c5
+        self.c6 = parameters.c6
+        self.c7 = parameters.c7
+        self.c8 = parameters.c8
+
+    def __call__(self, device, input_tensor):
+        input_tensor = input_tensor.to(device, self.c1.conv.input_sharded_memory_config)
+
+        output_tensor = self.c1(input_tensor)
```

Comment:
call the new conv api and provide reshard_if_not_optimal==True. you must provide the shard scheme height_sharding= True